app-config.json

```diff
@@ -763,6 +763,176 @@
             }
           ]
         },
+        {
+          "name": "Issue Object Event",
+          "id": "issue_object_event",
+          "components": [
+            {
+              "name": "JsonForm",
+              "type": "EntryData",
+              "props": {
+                "schema": {
+                  "url": "https://jargon.sh/user/unece/traceabilityEvents/v/0.4.0/artefacts/jsonSchemas/ObjectEvent.json?class=ObjectEvent"
+                },
```

Comment:
I noticed that the traceability events (TE) data model has been updated to v0.5.0, so the TE data model v0.4.0 seems a bit outdated. However, if we use a TE data model version higher than 0.3.10, the current render template might not align properly since it’s based on the design that matches TE data model 0.3.10.

You might want to consider whether to use the latest version or stick with the version that fits the current TE render template.

Reply:
Yep, that is an example app config, the processObjectEvent can work all the versions, so I think that is flexible to change.